### ISSUE TITLE
[feature-request] Is it possible to integrate this into compile time check?

### ISSUE THREAD

**Issue by Irvingouj** [kaleidawave/ezno#89]
for example, is sqlx, we can do compile time syntax check on sql. it would be fantastic if we can do the same for js/ts. This would be veryy useful in 
1. wasm_bindgen_test
where you could just write correct js code with syntax check to run your test. Allows for better CI testing.
2. wasm_bindgen(typescript_custom_section)
where you want to have customized type for the interfaces you provide.

**Comment by kaleidawave** [comment]
Yes, technically it is a standard Rust crate so you should be able to check JS/TS code in build.rs, macros etc. I currently wouldn't recommend doing it today while Ezno and the exposed API is still WIP. But in the future certainly!
